src/proxy/cache/itemPreloader.js

```diff
@@ -280,6 +280,10 @@ function itemPreloaderFactory(options) {
                     setItemFlag(item, 'hasFeedbacks');
                 }
 
+                if (_.size(item.portableElements.pci)) {
+                    item.flags = Object.assign({}, item.flags, { hasPci: true });
+                }
```

Comment:
**chore:** Meanwhile, helpers have been added to manage the flags.

**suggestion:**
```suggestion
                if (_.size(item.portableElements.pci)) {
                    setItemFlag(item, 'hasPci');
                }
```